db/migrate/20160610124506_add_lives_on_to_middleware_server.rb

```diff
@@ -0,0 +1,11 @@
+class AddLivesOnToMiddlewareServer < ActiveRecord::Migration[5.0]
+  def up
+    add_column :middleware_servers, :lives_on_type, :string
```

Comment:
oh, and for polymorphic relations you can use this nice
`t.references :lives_on, :type => :bigint`
which also adds an index.